src/GeneralStateTestsFiller/stMemoryTest/bufferFiller.yml

```diff
@@ -0,0 +1,442 @@
+             } NOP)
+           (if (= bufferType 13) { 
+                   [length] 0x1000 
+                   [offset] 0x0
+             } NOP)
+           ; bufferType 20 is offset 2^256-10, length 5
+           ; this causes an out of gas error because the gas cost
+           ; is around 2^251 (memory is priced in 32 byte chunks)
+           (if (= bufferType 20) { 
+                   [length] 5
+                   [offset] (- 0 10)
+             } NOP)
+
+           (if (= opcode 0x20) (sha3 @offset @length) NOP)
+           (if (= opcode 0x37) (calldatacopy @offset 0 @length) NOP)
+           (if (= opcode 0x39) (codecopy @offset 0 @length) NOP)
+           (if (= opcode 0x3C) (extcodecopy 0xc0de @offset 0 @length) NOP)
+           (if (= opcode 0x3E) (returndatacopy @offset 0 @length) NOP)
+           (if (= opcode 0xA0) (log0 @offset @length) NOP)
+           (if (= opcode 0xA1) (log1 @offset @length 1) NOP)
+           (if (= opcode 0xA2) (log2 @offset @length 1 2) NOP)
+           (if (= opcode 0xA3) (log3 @offset @length 1 2 3) NOP)
+           (if (= opcode 0xA4) (log4 @offset @length 1 2 3 4) NOP)
+           (if (= opcode 0xF0) (create 0 @offset @length) NOP)
+           (if (= opcode 0xF1)  (call 0x1000 0xc0de 0 @offset @length 0 0) NOP)
+           (if (= opcode 0x1F1) (call 0x1000 0xc0de 0 0 0 @offset @length) NOP)
+           (if (= opcode 0xF2)  (callcode 0x1000 0xc0de 0 @offset @length 0 0) NOP)
+           (if (= opcode 0x1F2) (callcode 0x1000 0xc0de 0 0 0 @offset @length) NOP)
+           (if (= opcode 0xF4)  (delegatecall 0x100000 0xc0de @offset @length 0 0) NOP)
+           (if (= opcode 0x1F4) (delegatecall 0x100000 0xc0de 0 0 @offset @length) NOP)
+           (if (= opcode 0xF5) (create2 0 @offset @length 0x5A17) NOP)
+           (if (= opcode 0xFA)  (staticcall 0x100000 0xc0de @offset @length 0 0) NOP)
+           (if (= opcode 0x1FA) (staticcall 0x100000 0xc0de 0 0 @offset @length) NOP)
+
+           ; RETURNDATACOPY, after making sure there is return data
+           (if (= opcode 0x13E) {
+               (call 0x1000 0xc0de 0 0 0 0x100 0x100)
+               (returndatacopy @offset 0 @length) 
+           } NOP)
+
+           ; check return, requires special handling
+           (if (= opcode 0xF3)  (call 0x100000 0xf30c0de 0 length 0x40 0 0) NOP)
+
+
+           ; check revert, requires special handling
+           (if (= opcode 0xFF)  {
+              (call 0x100000 0xff0c0de 0 length 0x40 0 0)
+              [[0]] (returndatasize)
+           } NOP)
+
+           ; Kill the goat
+           [[0x100]] 0
+        }
+      nonce: '0'
+      storage:
+        # Sacrificial goat to tell reverts apart from successes
+        0x100: 0x60A7
+
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: '0x'
+      nonce: '0'
+      storage: {}
+      
+  transaction:
+    data:
+    # SHA3       
+    - :label ok   :abi f(uint,uint) 0x20 0   # Valid buffer length
```

Comment:
what is the meaning of 0 1 2 10 11 12 12 ?

Reply:
Sorry, the last one was supposed to be 13. I fixed the comment.